tests/lean/run/hash_map1.lean

```diff
@@ -1,29 +0,0 @@
-import data.hash_map
```

Comment:
This test was removed because obviously the core doesn't have hash_maps anymore. Perhaps we could create a test directory in stdlib as well, but since `hash_map` is now fully verified the test cases seem moot.